gitlab/datadog_checks/gitlab/data/conf.yaml.example

```diff
@@ -1,7 +1,7 @@
 init_config:
```

Comment:
Since `allowed_metrics` is so long, let's move the `init_config` section below `instance` section for visibility on the updated config options. What do you think?

Reply:
Yeah that makes sense!